contribs/gmf/examples/search.js

```diff
@@ -0,0 +1,65 @@
+goog.provide('gmf-search');
+
+goog.require('gmf.mapDirective');
+goog.require('gmf.searchDirective');
+goog.require('ngeo.FeatureOverlayMgr');
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.layer.Tile');
+goog.require('ol.source.OSM');
+
+
+/** @const **/
+var app = {};
+
+
+/** @type {!angular.Module} **/
+app.module = angular.module('app', ['gmf']);
+
+
+
+/**
+ * @param {ngeo.FeatureOverlayMgr} ngeoFeatureOverlayMgr The ngeo feature
+ *     overlay manager service.
+ * @constructor
+ * @export
```

Comment:
I'll go ahead and create a PR. There's also another minor change I'd like to make.